src/Stripe.net/Entities/InvoiceLineItems/InvoiceLineItem.cs

```diff
@@ -20,33 +19,24 @@ public class InvoiceLineItem : StripeEntity<InvoiceLineItem>, IHasId, IHasMetada
         [JsonProperty("currency")]
         public string Currency { get; set; }
 
-        [JsonProperty("date")]
-        [JsonConverter(typeof(UnixDateTimeConverter))]
-        public DateTime Date { get; set; } = Stripe.Infrastructure.DateTimeUtils.UnixEpoch;
-
```

Comment:
make sure to flag this explicitly in the PR and the merge part because the property is entirely removed